.gitignore

```diff
@@ -4,4 +4,5 @@ build
 .env
 .DS_Store
 dist
-secrets/*.json
\ No newline at end of file
+secrets/*.json
+src/data/~*
```

Comment:
This folder won't change. Currently we don't accept external files. But it is on the board.